pac4j-saml/src/main/java/org/pac4j/saml/util/Configuration.java

```diff
@@ -0,0 +1,88 @@
+package org.pac4j.saml.util;
+
+import com.sun.org.apache.xerces.internal.util.SecurityManager;
+import net.shibboleth.utilities.java.support.component.ComponentInitializationException;
+import net.shibboleth.utilities.java.support.xml.BasicParserPool;
+import net.shibboleth.utilities.java.support.xml.ParserPool;
+import org.opensaml.core.config.ConfigurationService;
+import org.opensaml.core.config.InitializationException;
+import org.opensaml.core.config.InitializationService;
+import org.opensaml.core.xml.XMLObjectBuilderFactory;
+import org.opensaml.core.xml.config.XMLObjectProviderRegistry;
+import org.opensaml.core.xml.config.XMLObjectProviderRegistrySupport;
+import org.opensaml.core.xml.io.MarshallerFactory;
+import org.opensaml.core.xml.io.UnmarshallerFactory;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Misagh Moayyed
+ */
```

Comment:
Don't forget the `@since` annotation so we can track changes/improvments over time...